zephyr/module.yml

```diff
@@ -0,0 +1,2 @@
+build:
+  cmake: .
```

Comment:
you probably want something like

```
build:
  cmake-ext: True
  kconfig-ext: True
```

here? That way you can manage the cmake and kconfig in the upstream `module` directory